AirCasting/CoreData/CoreDataHook.swift

```diff
@@ -27,6 +27,19 @@ final class CoreDataHook: NSObject, ObservableObject {
             self.refreshSessions()
         }
     }
+    
+    func setup(selectedSection: DashboardSection) throws {
+        self.selectedSection = selectedSection
+        try? performFetch(for: selectedSection)
```

Comment:
Thinking out loud - I'm wondering if moving `try? performFetch(for: selectedSection)` to `refreshSessions()` wouldn't make sense - wdyt? 
Like, in the future, when using `refreshSessions()` you'd expect it to fetch sessions, wouldn't you?

Reply:
After all this `refreshSessions()` is used in few places and it makes it not so easy to change. I will give it a look in different PR

Reply:
didn't check that so good job :D